microsoftgraph/microsoft-graph-explorer-v4
# Fix: screen reader announces search results status message

## Overview

Added announced component which enables a screen reader to announce the 'Search sample queries' and 'Search permissions' (search result) information.

## Testing Instructions
* Use Microsoft Edge browser
* Ensure you have a screen reader turned on (e.g Windows Narrator)

1. Navigate to the 'Sample Queries' button by tab key and select it.
2. Navigate 'Sample Queries' search edit field and type 'Retrieve'.
3. Verify the screen reader's behavior. The screen reader announces search results after every letter typed.
